File: JAVA/Transpose.java
import java.lang.*;
public class Transpose{
    public static void main(String arg[])
    {
        int A[][] = {{1,2,3},{4,5,6},{7,8,9}};
        int B[][] = new int[3][3];
        System.out.println("Matrix:");
        for(int x[]:A)
        {
            for(int y:x)
            {
                System.out.print(y+" ");
            }
            System.out.println();
        }
        for(int i=0;i<A.length;i++)
        {
            for(int j=0;j<A[0].length;j++)
            {
                B[i][j]=A[j][i];
            }
        }
        System.out.println("Transposed Matrix:");
        for(int x[]:B)
        {
            for(int y:x)
            {
                System.out.print(y+" ");
            }
            System.out.println();
        }
    }
}

File: cpp/transpose.cpp
/*
    Authors Name : Mayuri Kolhe
    Date Modified: 1 October, 2022
*/


/* Program to find Transpose of a matrix */

#include <iostream>
using namespace std;
 
int main()
{
    static int array[10][10];
    int i, j, m, n;
 
    cout<<"Enter the order of the matrix \n";
    // Inputing elements in matrix from user
    cin>>m>>n;
    cout<<"Enter the coefiicients of the matrix\n";
    for (i = 0; i < m; ++i)
    {
        for (j = 0; j < n; ++j)
        {
            cin>>array[i][j];
        }
    }
    //Printing the original matrix
    cout<<"The given matrix is \n";
    for (i = 0; i < m; ++i)
    {
        for (j = 0; j < n; ++j)
        {
            cout<<" "<<array[i][j];
        }
        cout<<"\n";
    }
    //Printing the transpose of matrix
    cout<<"Transpose of matrix is \n";
    for (j = 0; j < n; ++j)
    {
        for (i = 0; i < m; ++i)
        {
           cout<<" "<<array[i][j];
        }
        cout<<"\n";
    }
    return 0;
}